### DISCUSSION TITLE
Speed Up type checking

### DISCUSSION THREAD

**Discussion by sheremet-va** [johnsoncodehk/vue-tsc#41] [Q&A]
Is there a way to speed up type checking? I know it is possible to use `incremental` with `tsc`, but `vue-tsc` doesn't suppport that yet. Are there any other ways?

Or maybe it's possible to turn on `incremental` only for `ts` files with some special flag?

**Comment by johnsoncodehk** [comment]
`incremental` support is possible, but there is a lot of backlog at the moment... I can't say when to support it.
